src/main/java/com/schematica/client/renderer/chunk/overlay/RenderOverlay.java

```diff
@@ -178,32 +177,33 @@ private static void increamentRenderChunkedUpdated()
     private static int getSides(final Block block, final World world, final BlockPos pos, int s)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "block". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1172)